pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -123,47 +135,54 @@ func (r *ReplicaSetReconciler) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	r.log.Debug("Building service")
 	svc := buildService(mdb)
 	if err = r.client.CreateOrUpdate(&svc); err != nil {
 		r.log.Infof("The service already exists... moving forward: %s", err)
 	}
 
+	r.log.Debug("Creating/Updating StatefulSet")
```

Comment:
same comment as above

Reply:
this is indicated by the zap context `r.log = zap.S().With("ReplicaSet", request.NamespacedName)` at the beginning of the function.